packages/spectral/src/types/ComponentRegistry.ts

```diff
@@ -45,7 +45,8 @@ type ComponentReferenceTypeValueMap<
     actions: ValueExpression<TValue>;
     triggers: ValueExpression<TValue> | ConfigVarExpression;
     dataSources: ValueExpression<TValue> | ConfigVarExpression;
-    connections: (ValueExpression<TValue> | ConfigVarExpression) & ConfigVarVisibility;
+    connections: (ValueExpression<TValue> | ConfigVarExpression) &
+      ConfigVarVisibility & { writeOnly?: true };
```

Comment:
This seems wrong to me.  Connections themselves aren't marked as writeOnly, it is the individual inputs on a connection that get marked as writeOnly.

Reply:
This type refers to the `connections.values` block, not the connection itself.

![Screenshot 2024-10-17 at 2 31 46 PM](https://github.com/user-attachments/assets/e3d949b1-3b50-4d1d-9506-942eec568acd)

Hinting examples:
![Screenshot 2024-10-17 at 2 35 59 PM](https://github.com/user-attachments/assets/e26cc31b-b620-4eff-879b-55a9bf4a98e8)
![Screenshot 2024-10-17 at 2 37 00 PM](https://github.com/user-attachments/assets/6e790a28-1e68-41f5-ae0b-e2a7ebbabd93)